index.js

```diff
@@ -12,6 +13,9 @@ const readFile = promisify(fs.readFile);
 const writeFile = promisify(fs.writeFile);
 
 const copyWithTemplate = async (from, to, variables) => {
+	const dirname = path.dirname(to);
+	await makeDir(dirname);
```

Comment:
Why is this needed now?

Reply:
`copyWithTemplate` is not isuing `cpy` for copying files.

`cpy` can make directory automatically, but the `copyWithTemplate` is not.